portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,43 +27,70 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages;
+  private class UserComment {
+    String message;
+    String userEmail;
```

Comment:
These have the default access specifier (ie no `public` or `private`) which means that anything in this file can access them. Do you want them to be private?